vlib/v/checker/tests/match_expr_with_none_only.out

```diff
@@ -0,0 +1,12 @@
+vlib/v/checker/tests/match_expr_with_none_only.vv:2:2: warning: unused variable: `a`
+    1 | fn main() {
+    2 |     a := match 1 {
+      |     ^
+    3 |         1 { none }
+    4 |         else { none }
+vlib/v/checker/tests/match_expr_with_none_only.vv:2:7: error: invalid if expression, must supply at least one value other than `none`
```

Comment:
```suggestion
vlib/v/checker/tests/match_expr_with_none_only.vv:2:7: error: invalid match expression, must supply at least one value other than `none`
```

Reply:
Oh, I've fixed on same time than you. Thanks for reviewing. 👍🏻